aws-rollback.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+
+import json
+import subprocess
+import sys
+
+DEFAULT_REPOSITORY_NAME = "rust-central-station"
+TARGET_TAG = 'latest'
+ECS_CLUSTER = 'rust-ecs-prod'
+ECS_SERVICE = 'TODO COMPLETE'
+
+def main():
+    """CLI entrypoint of the program."""
+    repository_name = DEFAULT_REPOSITORY_NAME if len(sys.argv) < 2 else sys.argv[1]
```

Comment:
I'd prefer not to have a default repository name.

Reply:
Great, just to clarify the `repository name` should be a required argument.

Reply:
Yes.